utils/config-utils.sh

```diff
@@ -253,7 +272,10 @@ config::generate_service_configs() {
     local -r TARGET_FOLDER_NAME=$(basename "${TARGET_FOLDER_PATH}")
     local count=0
 
-    try "touch ${COMPOSE_PATH}/docker-compose.tmp.yml" "Failed to create temp service config compose file"
+    try \
+        "touch ${COMPOSE_PATH}/docker-compose.tmp.yml" \
+        catch \
+        "Failed to create temp service config compose file"
```

Comment:
If this fails it will be a config importing fail, which is a significant deploy fail. Please do try-throw.

Reply:
cool, done in [a0370df](https://github.com/jembi/platform/pull/188/commits/a0370dfd3a72d6c29352623304555257bccbe370)